connectors/zapier/utils/__init__.py

```diff
@@ -91,6 +91,7 @@
     'sellv2': 'hooks.zapier.com/hooks/catch/2517134/o356ehs/',
     'tax': 'hooks.zapier.com/hooks/catch/2517134/odq2hzt/',
     'ratingv2': environ['HOOK_RATING_V2'],
+    'rating_v2_commerzbank': 'hooks.zapier.com/hooks/catch/3188987/b3m3l72/',
```

Comment:
New hook, it will be fetched in else hook.

Reply:
Is it a secret though?

Reply:
technically yes - practically no. Because endpoint proxying data to here is also not protected …